console/src/components/CompactHeader/CompactHeader.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import kind from '@enact/core/kind';
+import Button from '@enact/agate/Button';
+
+import css from './CompactHeader.less';
+import {getPanelIndexOf} from '../../App';
+
+const CompactHeader = kind({
+	name: 'CompactHeader',
+
+	propTypes: {
+		children: PropTypes.node,
+		isFor: PropTypes.string,
+		noExpandButton: PropTypes.bool,
+		onExpand: PropTypes.func
+	},
+
+	styles: {
+		css,
+		className: 'compactHeader',
+		publicClassNames: true
+	},
+
+	defaultProps: {
+		noExpandButton: false
+	},
+
+	handlers: {
+		onTabChange: (ev, {isFor, onExpand}) => {
+			const tabIndex = getPanelIndexOf(isFor);
```

Comment:
I'm a little concerned about the coupling this function introduces. Does the header need to understand that there is a mapping between names and indices? Would it be better for the event to only include the name and let the consumer (the App) to map that to whatever it needs?